src/components/item-display-panel/item-display-panel.component.ts

```diff
@@ -39,17 +39,21 @@ export class ItemDisplayPanelComponent {
     }
 
     @Input()
+    get footer() {
+        return this._footer;
+    }
+
+    set footer(footer: boolean) {
+        this._footer = typeof footer === 'string' ? !(footer === 'false') : footer;
+    }
+
+    @Input()
     get boxShadow() {
         return this._boxShadow;
     }
 
     set boxShadow(boxShadow: boolean) {
-        let value: any = boxShadow;
-        if (value === 'false') {
-            this._boxShadow = false;
-        } else {
-            this._boxShadow = true;
-        }
+        this._boxShadow = typeof boxShadow === 'string' ? !(boxShadow === 'false') : boxShadow;
```

Comment:
Could `!(boxShadow === 'false')` be changed to `boxShadow === true` or even `boxShadow !== 'false'` to simplify the logic?